wayfire.ini

```diff
@@ -151,15 +197,25 @@ idle = swayidle before-sleep swaylock
 
 # Start a terminal
 # https://github.com/alacritty/alacritty
-binding_terminal = <super> KEY_ENTER
+binding_terminal = <super> KEY_ENTER | <ctrl> <alt> KEY_T
 command_terminal = alacritty
 
 # Start your launcher
 # https://hg.sr.ht/~scoopta/wofi
 # Note: Add mode=run or mode=drun to ~/.config/wofi/config.
 # You can also specify the mode with --show option.
 binding_launcher = <super> <shift> KEY_ENTER
-command_launcher = wofi
+command_launcher = wofi --show=run
+
+# sgtk-grid
```

Comment:
I think we spoke about this, you could create a section in the wiki for this, I do not think it belongs to the default config.